lib/python/picongpu/plugins/plot_mpl/compare_1d_visualizer.py

```diff
@@ -0,0 +1,185 @@
+"""
+This file is part of the PIConGPU.
+
+Copyright 2017-2018 PIConGPU contributors
+Authors: Sophie Rudat
+License: GPLv3+
+"""
+
+from picongpu.plugins.compare import CompareData
+from picongpu.plugins.plot_mpl.base_visualizer import Visualizer as\
+    BaseVisualizer, plt
+import numpy as np
+
+
+class Visualizer(BaseVisualizer):
```

Comment:
Please check the updated interface of the `base_visualizer.py` and refactor the code in a similar way as suggested for your PR #2737